ariadne/executable_schema.py

```diff
@@ -65,3 +71,15 @@ def repair_default_enum_values(schema, bindables) -> None:
     for bindable in bindables:
         if isinstance(bindable, EnumType):
             bindable.bind_to_default_values(schema)
+
+
+def set_outname_to_snake_case(schema) -> None:
+    for graphql_type in schema.type_map.values():
+        if not isinstance(graphql_type, (GraphQLObjectType, GraphQLInputObjectType)):
```

Comment:
We should do `if isinstance(graphql_type, GraphQLInputObjectType)` and then set field.out_name, then `if isinstance(graphql_type, GraphQLObjectType)` do that for args.